debian/libjxl-gimp.install

```diff
@@ -0,0 +1 @@
+usr/lib/gimp/2.0/plug-ins/file-jxl/file-jxl 
```

Comment:
This path comes from `gimp-2.0` pkg-config. I guess it is fine to hard-code it here since we hard-code the gimp-2.0? I don't know what's the common thing to do.

Reply:
I just looked in another plugin, and they just have `/usr/lib/gimp`.  I'll change it.